lib/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -714,6 +717,26 @@ namespace Opm {
         }
     }
 
+    void Schedule::handleWTEMP( const DeckKeyword& keyword, size_t currentStep) {
+        for( const auto& record : keyword ) {
+            const std::string& wellNamePattern = record.getItem("WELL").getTrimmedString(0);
+
+            for (auto* well : getWells(wellNamePattern)) {
+                // TODO: Can this be done like this? Setting the temperature only has an
+                // effect on injectors, but specifying this for producers won't hurt and
+                // wells can also switch their injector/producer status. Note that
```

Comment:
Regarding this comment block: As I understand it there is an undetonated bomb due to: "Note modifying the injector properties for producer wells currently leads to a very weird segmentation fault downstream." - I guess that should be understood/fixed before this is merged?

Reply:
@GitPaean, @totto82, @atgeirr: someone of you probably needs to have a go at this. as far as I understood is that the well controls code gets confused if production wells have injection properties. That said, WTEMP is likely not the only keyword which only makes sense for injection wells, so this kind of problem is probably already present in the current opm-parser code in other contexts.